### ISSUE TITLE
Can't load checkpoint to continue training

### ISSUE THREAD

**Issue by 1160300911** [salesforce/TabularSemanticParsing#9]
Hi, I encountered a bug when the system finished an epoch from a checkpoint. 

The original error message may be a little confusing. Sorry about that. I found that when the training reaches about the 9000 step, the loss will become 'nan'. Compared with the original parameters, due to the 12GB limit, I changed the 'train_batch_size' and 'num_accumulation_steps', in order to ensure that the full batch size is equal to 32.

----------------------------------------------------------------------------------------------------------------------------------------------
Supplement: I'm sorry I forgot to provide the changed parameters. I made two changes in total:

num_accumulation_steps: 2 -> 16
train_batch_size: 16 -> 2

In addition, I cloned the latest version of the code yesterday, and once again encountered the problem of loss becoming ‘nan’. The error message is as followed:

```
Step 6999.9375: average training loss = 0.02193380185714989
0 pre-computed prediction order reconstruction cached
100%|██████████| 345/345 [28:50<00:00,  3.37s/it]
Dev set performance:
Top-1 exact match: 0.6334622823984526
Top-3 exact match: 0.7272727272727273
 73%|███████▎  | 11648/16000 [1:58:43<39:58,  1.81it/s]> /home/rxsun/TableQA/TabularSemanticParsing/src/common/nn_modules.py(695)forward()
-> if loss > 1e8:
(Pdb) l
690                 return 0
691             loss = F.nll_loss(masked_inputs, masked_targets)
692             if torch.isnan(loss):
693                 import pdb
694                 pdb.set_trace()
695  ->         if loss > 1e8:
696                 import pdb
697                 pdb.set_trace()
698             return loss
699
700
(Pdb) loss
tensor(nan, device='cuda:0', grad_fn=<NllLossBackward>)
```

Could you give me some advice about that？Thanks!
